src/api/firebase.js

```diff
@@ -166,7 +167,9 @@ export async function updateItem(listId, item) {
 	}
 }
 
-export async function deleteItem() {
+export async function deleteItem(listId, id) {
+	const docRef = doc(db, listId, id);
+	await deleteDoc(docRef);
```

Comment:
What would happen if this tries to delete a document that doesn't exist? Meaning that the `itemId` is not in the collection?

Reply:
So I tried this with two tabs open locally... when I deleted the item in one tab it was also removed from the second tab. I guess it re-rendered automatically?...